Source/Modules/java.cxx

```diff
@@ -4387,7 +4387,7 @@ class JAVA:public Language {
       for (p = l; p;) {
 	if ((tm = Getattr(p, "tmap:directorargout"))) {
 	  addThrows(n, "tmap:directorargout", p);
-	  Replaceall(tm, "$result", "jresult");
+	  Replaceall(tm, "$result", Getattr(p, "name"));
```

Comment:
It looks like a similar change is required in plenty of the other languages, like csharp.cxx.